### ISSUE TITLE
Successfully uninstalled Boxstarter via Chocolatey -- update policies still in place

### ISSUE THREAD

**Issue by Newbie13XD** [chocolatey/boxstarter#383]
### What You Are Seeing?
The update policies Boxstarter put on my PC during the Node.js install persist even though Boxstarter was uninstalled and my PC promptly was rebooted.

### What is Expected?
I never expected these update policies in the first place, and I expected uninstalling Boxstarter to remove them. 

### How Did You Get This To Happen? (Steps to Reproduce)
1. Install Node.js 10.13.0 via the MSI. 
2. Choose to install Chocolatey and Boxstarter
3. Remove the Boxstarter packages using choco uninstall <pkg names>

**Comment by pauby** [comment]
When you say 'update policies' are you referring to Windows Update?

**Comment by Newbie13XD** [comment]
Here is what I am seeing. 

![image](https://user-images.githubusercontent.com/4428753/50576418-9c3d3280-0e11-11e9-8ac6-fcf4fe8f6c0c.png)
![image](https://user-images.githubusercontent.com/4428753/50576421-a8c18b00-0e11-11e9-8227-a7b12b0ee8f8.png)

**Comment by Newbie13XD** [comment]
Yes, indeed, I have read through that thread but I failed to find a solution! I had the same experience as runewake2, where the commands didn't seem to help much (this was prior to me uninstalling Boxstarter) and gpedit.msc, well, I attempted that method as my initial instinct when this happened, but I will attach a screenshot showing you exactly what I'm seeing in case I am missing something.

![image](https://user-images.githubusercontent.com/4428753/50576597-7c0f7280-0e15-11e9-9ea4-d98552e32648.png)

**Comment by pauby** [comment]
Not finding a solution in the other issue doesn't warrant opening a new one. This is not a new issue. It's the same issue.

Please use the other issue so we can keep all related issues in the one place.

**Comment by Newbie13XD** [comment]
You do have a very good point there. My apologies.
